src/User.cpp

```diff
@@ -74,7 +62,9 @@ void User::closeSocket() {
 
 int User::getSocket() const { return this->socket; }
 
-const string &User::getNickname() const { return this->nickname; }
+string User::getNickname() const {
```

Comment:
again dit moet je niet veranderen in een van de meest gebruikte functies in heel de code waardoor nu iedere keer dat deze gecalled wordt een kopie van een string gemaakt wordt.